### ISSUE TITLE
Add documentation/wiki entry for technologies

### ISSUE THREAD

**Issue by St4NNi** [FAIR-DS4NFDI/wiki#2]
This is an issue to collect ideas what could be documented in the "technologies" section:

1. Expand and refine the information from here: https://github.com/FAIR-DS4NFDI/wiki/wiki/WP3-WP4-Technical-Foundations-and-Demonstrators 

- [ ] Cloud infrastructure
- [ ] CI/CD infrastructure
- [ ] Base Storage
- [ ] Storage Metadata management
- [ ] EDC
- [ ] OIDC ? -> might be too vague

Especially focus on how they fit in a "FAIR-DS" architecture.

**Comment by hk-accurids** [comment]
It would be great to have a stepwise guideline of how to exchange data following the IDS protocol, including example JSON messages. E.g.
1. Create asset
 Example POST
 Example RESPONSE
2. Create policy
etc

**Comment by hk-accurids** [comment]
What about a section about recommended documentation?
Like a "read this first". It could help the next contributors to onboard more efficiently.

**Comment by St4NNi** [comment]
> It would be great to have a stepwise guideline of how to exchange data following the IDS protocol, including example JSON messages. E.g.
> 
>     1. Create asset
>        Example POST
>        Example RESPONSE
> 
>     2. Create policy
>        etc

Yes, but we should reference the official documentation for this, just to be sure that it is as up-to-date as possible.

**Comment by hk-accurids** [comment]
@St4NNi I have added a page about the EDC connector with example messages. The documentation renders well in Github but it does not yet work well with mbook. Could you please have a look?

**Comment by St4NNi** [comment]
> @St4NNi I have added a page about the EDC connector with example messages. The documentation renders well in Github but it does not yet work well with mbook. Could you please have a look?

The mdbook automatic translation to `.html` files will only work if the markdown file is part of the `SUMMARY.md` i shouldve fixed it.

**Comment by hk-accurids** [comment]
Perfect! thank you @St4NNi
